### PR TITLE
Mod rebalance

### PR DESCRIPTION
**Pull request by OneEyeMaker**
Removed included Thaumcraft API (use dependency instead)
Updated to Forge 1614
Rebalanced mod recipes.

@Dream-Master 
What I changed:
Earlier recipe wad _very_ unefficient (author used TMaps with strings for storing aspects and parsing them later). Now everything uses native Thaumcraft's AspectList.

Earlier this mod allowed to craft Attuned node (with up to 8 aspects) and infuse in to Node in a Jar with up to 8 * 15 (= 120) aspects of one type. Spend *only 8 phials* (= 64 essentia, *not a big deal*) in crafting and get node with 120 aspects (and spent some more essentia for infusion) - it's cheat.

**_Now_** this mod allow to craft nodes of **any** size (from *two* aspects to *thousands*). Player just add phials with essentia into crafting grid with matrix. Each phial adds 2 aspects (each phial stores 8 essentia, i. e. 8 essentia => 2 vis). And main feature - **this craft can be repeated multiple times**. In other words, player can add more and more essentia to get larger node.
To infuse this attuned node to Node in a Jar, player needs **the same** amount of essentia, which he spent on craft.
In summary: 8 essentia for 1 vis in node.

Please, don't merge this PR now: I want to change matrix recipe. For me now it's too hard and too late. Because cost of node now isn't cost of matrix itself, but cost of it's attunement (creation of node), I suggest to change recipe to end-MV age - start HV age (when aura nodes very needed). 
Any recipe suggestionsare welcome.

### CODE REVIEW

**Comment by Dream-Master** [comment]
@OneEyeMaker matrix ? did you mean Thaumcraft matrix or the matrix from Nodal machanics?

**Comment by OneEyeMaker** [comment]
@Dream-Master I'm speaking about this mod.
Now matrix (for Aura Node creation) requires Altar from Blood Magic tier 5 (!) to craft (demonic shard in recipe). For me it's too late (at that point I probably will have transposer from BM for teleporting all nodes)

Just download and test this version. You'll see - Unattuned Matrix shouldn't cost so much, because Aura Node craft from this item costs so.

**Comment by Dream-Master** [comment]
Ok so what you put inside now as a replace item ?

**Comment by OneEyeMaker** [comment]
I remade recipe for Unattuned Matrix.
Like that:


I also bumped warp amount for this matrix. Players will gat this warp with large amounts, when they'll craft this item. It is like *taboo*
I think, that it's final recipe and PR ready for merge.

**Comment by Dream-Master** [comment]
@OneEyeMaker thanks
